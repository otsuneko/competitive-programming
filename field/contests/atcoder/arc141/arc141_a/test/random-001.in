41
937956674
89523943
170805760
809335233
160601178
184158275
771113196
399551833
175774529
60440533
513107370
421931318
687576190
35044597
815126884
997520076
517916321
304046659
481039458
1213034
215534231
190331200
43967830
180229494
119593041
772346423
88390927
135955228
33715906
904208442
889707860
39707442
603692594
705072402
881891205
235940824
954660820
829445824
908449185
915545818
592638941
