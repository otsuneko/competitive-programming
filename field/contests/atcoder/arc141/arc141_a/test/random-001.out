937937937
89518951
170170170
808808808
160160160
183183183
770770770
399399399
175175175
60436043
512512512
421421421
686686686
35043504
814814814
996996996
517517517
303303303
480480480
1111111
215215215
190190190
43964396
180180180
119119119
771771771
88388838
135135135
33713371
903903903
888888888
39703970
603603603
704704704
881881881
235235235
953953953
828828828
907907907
914914914
592592592
